aii-opennebula/src/main/perl/opennebula.pod

```diff
@@ -96,8 +99,6 @@ All these paths are relative to /system/aii/hooks/remove
 
 Stop running VM
 
-=over 4
```

Comment:
hmm, don't we validate pods in AII tests?

```code```
[wdpypere@scyther ~]$ podchecker opennebula.pod 
*** ERROR: =back without =over at line 112 in file opennebula.pod
opennebula.pod has 1 pod syntax error.
```